dotcms-integration/src/test/java/com/dotmarketing/portlets/browser/ajax/BrowserAjaxTest.java

```diff
@@ -229,13 +235,14 @@ public void testFolderTreeInfoWithInvalidFolderId() throws Exception {
     /**
      * Sets up the DWR context using Mockito so that the {@link BrowserAjax} class can be tested correctly.
      */
-    private static void setUpDwrContext() {
+    private static void setUpDwrContext(User user) throws DotDataException, DotSecurityException {
```

Comment:
what's the user used for here? I don't see any uses. is it used at all ?

Reply:
Initially, this method was designed for logging in as a system user. However, I need to test my modifications as a limited user instead of a system user. Therefore, to avoid duplicating the code, I am adding the option of specifying the desired user as a parameter.